doc/quick-start/quick_start_tensorflow_1.15.md

```diff
@@ -20,37 +20,38 @@ under the License.
 # Quick Start
 
 This tutorial provides a quick introduction to using Deep Learning on Flink. 
-This guide will show you how to download the latest stable version of Deep
-Learning on Flink, install. You will run a simple Flink job locally to train
+This guide will show you how to download and install the latest stable version 
+of Deep Learning on Flink. You will run a simple Flink job locally to train
 a linear model.
 
 ## Environment Requirement
 
 - Java: 8
 - Python: 3.7 
 - Flink: 1.14
-- TensorFlow: 1.15.x or 2.3.x
+- TensorFlow: 1.15
 
 ## Download & Install
 
 ### Download Flink
-[Download the latest binary release](https://flink.apache.org/downloads.html) 
-of Flink 1.14, then extract the archive:
+Download a stable release of Flink 1.14, then extract the archive:
 
 ```sh
-tar -xzf flink-*.tgz
+curl -LO https://dlcdn.apache.org/flink/flink-1.14.3/flink-1.14.3-bin-scala_2.11.tgz
+tar -xzf flink-1.14.3-bin-scala_2.11.tgz
 ```
 
 Please refer to [guide](https://nightlies.apache.org/flink/flink-docs-release-1.14//docs/try-flink/local_installation/) 
 for more detailed step of downloading or installing Flink.
 
 ### Download Deep Learning on Flink
-You can download the binary release of Deep Learning on Flink from
-[release](https://github.com/flink-extended/dl-on-flink/releases), then extract
-the archive:
+You can download the stable released binary release of Deep Learning on Flink,
+then extract the archive:
 
 ```sh
-tar -xzf flink-ml-*.tgz
+curl -LO https://github.com/flink-extended/dl-on-flink/releases/download/0.4.0/flink-ml-dist-0.4.0-bin.tgz
```

Comment:
The tgz file is named according to the maven project module names. I'd like to rename all to modules to something like dl-on-flink-dist, dl-on-flink-framework, etc. Since it is a big refactor that will touch many files, I think it is better to make the change in another pr.

Reply:
Sounds good. Let's rename it in a separate PR.